app/controllers/dashboard_controller.rb

```diff
@@ -17,6 +21,24 @@ def index
     end
   end
 
+  def autoyast
+    @controller_node = Socket.gethostname
+    begin
+      suse_connect_config = Rails.cache.fetch("SUSEConnect_config") do
+        Velum::SUSEConnect.config
+      end
+      @suse_smt_url = suse_connect_config.smt_url
+      @suse_regcode = suse_connect_config.regcode
+      @do_registration = true
+    rescue Velum::SUSEConnect::MissingRegCodeException,
+           Velum::SUSEConnect::MissingCredentialsException
+      @do_registration = false
+    end
+    render "autoyast.xml.erb", layout: false, content_type: "text/xml"
+  rescue Velum::SUSEConnect::SCCConnectionException
+    render status: 503, nothing: true
```

Comment:
nothing: true [will be deprecated](http://stackoverflow.com/questions/34688726/the-nothing-option-is-deprecated-and-will-be-removed-in-rails-5-1). Maybe `head: 503` ?